ssis-vs2022: DotNet4.6-TargetPack dependency breaks internalization due to HTTP + missing checksum

Hello,

The `ssis-vs2022` package fails during Chocolatey for Business internalization due to its dependency on `DotNet4.6-TargetPack`.

This ultimately pulls in `DotNet4.6`, which:
- Uses an HTTP download URL, and  
- Does not provide a checksum  

As a result, the package cannot be internalized because empty checksums are no longer permitted for non-secure sources.

I’ve tested installing `ssis-vs2022` without this dependency, and it completed successfully on a clean VM. Given that Visual Studio 2022 requires .NET Framework 4.8 (which is already included in supported Windows versions), this dependency appears to be redundant.

Would you be able to remove the `DotNet4.6-TargetPack` dependency and republish the package?

Thanks


```
[1/3] ssis-vs2022  2.0.0 → 2.2.0
  Downloading from community...
Chocolatey v2.7.0 Business
Downloading existing package(s) to D:\a\_temp\ChocoPackages\ssis-vs2022\download

DotNet4.6 v4.6.81.20150925 [Approved]
Found internalizable Chocolatey functions. Inspecting values for remote resources.
Found 'http://download.microsoft.com/download/C/3/A/C3A5200B-D33C-47E9-9D70-2F7C65DAAD94/NDP46-KB3045557-x86-x64-AllOS-ENU.exe' for replacement.

Download of 'NDP46-KB3045557-x86-x64-AllOS-ENU.exe' completed.
WARNING: Missing package checksums are not allowed (by default for HTTP/FTP,
 HTTPS when feature 'allowEmptyChecksumsSecure' is disabled) for safety
 and security reasons. Although we strongly advise against it, if you
 need this functionality, please set the feature 'allowEmptyChecksums'
 ('choco feature enable -n allowEmptyChecksums') or pass in the option
 '--allow-empty-checksums'. See 'choco download -?' for details.
The integrity of the file 'NDP46-KB3045557-x86-x64-AllOS-ENU.exe'
 from 'http://download.microsoft.com/download/C/3/A/C3A5200B-D33C-47E9-9D70-2F7C65DAAD94/NDP46-KB3045557-x86-x64-AllOS-ENU.exe'
 has not been verified by a checksum in the package scripts.
For the question below, you have 30 seconds to make a selection.
Do you wish to allow the download to continue (NOT RECOMMENDED)?([Y]es/[[N]o]): 
Encountered error downloading package ssis-vs2022 v2.2.0:
Empty checksums are no longer allowed by default for non-secure sources.
 Please ask the maintainer to add checksums to this package. In the meantime
 if you need this package to work correctly, please enable the
 feature allowEmptyChecksums, or provide the runtime
switch '--allow-empty-checksums'. It is strongly advised against allowing
 empty checksums for non-internal HTTP/FTP sources.
  ##[error]Download failed for ssis-vs2022
```

--- a/automatic/ssis-vs2022/readme.md
+++ b/automatic/ssis-vs2022/readme.md
@@ -1,14 +1,12 @@
 # [<img src="https://cdn.jsdelivr.net/gh/bbtsoftware/chocolatey-packages@58c2f838f653b03f0b017c2cf36bcea22cf581ce/icons/ssis-vs2022.png" height="48" width="48" />Microsoft SQL Server Integration Services Projects](https://community.chocolatey.org/packages/ssis-vs2022)
 
-Extension for Visual Studio 2022 to build high performance data integration and workflow solutions, including extraction, transformation, and loading (ETL) operations for data warehousing.
+Extension for Visual Studio 2022/2026 to build high performance data integration and workflow solutions, including extraction, transformation, and loading (ETL) operations for data warehousing.
 
-Visit https://techcommunity.microsoft.com/t5/SQL-Server-Integration-Services/bg-p/SSIS for the latest information, tips, news, and announcements about SSIS directly from the product team.
+Visit https://techcommunity.microsoft.com/blog/ssis/sql-server-integration-services-projects-2022-ga/4507575 for the latest information, tips, news, and announcements about SSIS directly from the product team.
 
 ## Notes
 
 This package has the following additional requirements:
 
-* Visual Studio core editor
-* SQL Server Data Tools
-* .NET Framework 4 targeting pack
-* .NET Framework 4.5 targeting pack
+* Visual Studio 2022/2026
+* SQL Server Data Tools (SSDT)

--- a/automatic/ssis-vs2022/ssis-vs2022.nuspec
+++ b/automatic/ssis-vs2022/ssis-vs2022.nuspec
@@ -20,17 +20,12 @@ Visit https://techcommunity.microsoft.com/t5/SQL-Server-Integration-Services/bg-
 
 This package has the following additional requirements:
 
-* Visual Studio core editor
-* SQL Server Data Tools
-* .NET Framework 4 targeting pack
-* .NET Framework 4.5 targeting pack
+* Visual Studio 2022/2026
+* SQL Server Data Tools (SSDT)
 ]]></description>
     <tags>microsoft integration integration-services sql ssis ssis-vs2022 ssdt sql-server data-tools</tags>
-    <dependencies>
-      <dependency id="DotNet4.6-TargetPack" />
-    </dependencies>
   </metadata>
   <files>
     <file src="tools\**" target="tools" />
-  </files>  
+  </files>
 </package>

